.travis.yml

```diff
@@ -28,8 +28,10 @@ env:
   - DB=postgres
 
 install:
-  - sh -e ci/install.sh
+  - sh -e ci/apache/install.sh
```

Comment:
Could we cycle through the valid auth types? Want to be sure we don't let `database` and `none` become bitrotted.